lib/ret_web/controllers/page_controller.ex

```diff
@@ -462,14 +478,35 @@ defmodule RetWeb.PageController do
     {_, available_integrations_script} =
       Ret.Meta.available_integrations_meta() |> generate_config("AVAILABLE_INTEGRATIONS")
 
+    app_name = app_config["translations"]["en"]["app-name"] || RetWeb.Endpoint.host()
+    scene = hub.scene || hub.scene_listing
+    name = cond do
+      hub.name && scene && scene.name && scene.name != hub.name -> join_smart([hub.name, scene.name])
+      hub.name -> hub.name
+      scene && scene.name -> scene.name
+      true -> "a room on " <> app_name
```

Comment:
Do you have an example where the room name isn't present?

Reply:
The only way I know for `hub.name` to be nil is to edit the database directly, but I'm reluctant to presume that `hub.name` is non-nil.
This whole PR was kicked off by finding that `hub.scene` could be nil, and the page wouldn't load, so I'm coding very conservatively.

We could omit line 486 `scene && scene.name -> scene.name`, and if the room name is nil, the metadata would show "a room on foo" as the room name.